src/compiler/checker.ts

```diff
@@ -29587,7 +29602,7 @@ namespace ts {
             }
             else {
                 // Only here do we need to check that the initializer is assignable to the enum type.
-                checkTypeAssignableTo(checkExpression(initializer), getDeclaredTypeOfSymbol(getSymbolOfNode(member.parent)), initializer, /*headMessage*/ undefined);
+                checkTypeAssignableTo(checkExpression(initializer), getDeclaredTypeOfSymbol(getSymbolOfNode(member.parent)), initializer);

```

Comment:
??? Why was the parameter removed here? AFAIK passing `/*headMessage*/ undefined` disables error reporting, no?

Reply:
not passing it also disables error reporting. I ran a cleanup regex and it applied to this usage too. If you want to keep it to readability, I can put it back.

Reply:
Nah, it's fine.